packages/components/src/DateTimePickers/shared/InputSizer/InputSizer.component.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { format } from 'date-fns';
+
+const inputTextSizerStyle = {
+	fontWeight: 400,
+	fontSize: '16px',
+	visibility: 'hidden',
+};
+
+const placeholderSizerStyle = {
+	...inputTextSizerStyle,
+	fontStyle: 'oblique',
+};
+
+function InputSizer({ placeholder, inputText, children }) {
```

Comment:
Why do we need that ?
Why don't we have an input with a min width that would be big enough to contain the date ?